code/unclassified/minSubarraySizeWithDegree/minSubarraySizeWithDegree.cpp

```diff
@@ -0,0 +1,66 @@
+#include <bits/stdc++.h>
+#include <unordered_map>
+
+using namespace std;
```

Comment:
Don't forget to add the required comment as stated in the [contribution guidelines](https://github.com/OpenGenus/cosmos/wiki/contribute).